xml/schema/Contribute/Contribution.xml

```diff
@@ -368,6 +368,7 @@
     <readonly>true</readonly>
     <html>
       <label>Recurring Contribution</label>
+      <type>Number</type>
```

Comment:
Typically with FK fields the input type would be `EntityRef` - any reason not to do that here?

Reply:
Nope!